README.md

```diff
@@ -158,6 +159,33 @@ $ java -jar xpanse-runtime-1.0.0-SNAPSHOT.jar --spring.profiles.active=zitadel \
 --authorization-swagger-ui-client-id=221664356859969539@eclipse-xpanse
 ```
 
+##### Execute authenticated APIs
+
+When the application activated authorization with 'zitadel', execute authenticated APIs must add a header
+'Authorization' with value like follows 'Bearer ${access token}' in the http request. Here are two ways to get
+access_token for executing authenticated APIs:
+
+The first way, open the openAPI page 'http://localhost:8080/swagger-ui/index.html' of the ran Xpanse service in browser.
+![Browser open openAPI page](static/browser_open_openAPI_page.png)
+
+Click on the 'Authorize' icon on the OpenAPI page to open the authentication pop-up window. Click on the 'select all'
+option in the options behind the 'Scopes' on the authentication pop-up window, then click 'Authorize'.
+
+![Authorize in the authentication window](static/authorize_in_authorization_window.png)
+The browser will redirect to the login page of the IAM service. Fill username and password to complete user login. After
+successful login in the IAM the browser will back to the OpenAPI page with authorized info. Close the authentication
+window, before the authentication automatically expires or actively logout, you can select the API which you want to
+execute and click to expand it, then click on 'Try it out' and 'Execute' to execute the API you selected. In the Curl
+command, you can see the request header named 'Authorization' with the value of ${access_token} after prefix 'Bearer '.
+![Execute APIs using openAPI page](static/execute_api_in_openAPI_page.png)
```

Comment:
The browser will redirect to the login page of the IAM service. Fill username and password to complete user login. Once the login is successful, the control is automatically redirected to the Swagger-UI page. Close the authentication
window and select the API which you want to
execute and click to expand it, then click on 'Try it out' and 'Execute' to execute the API method. In the Curl
command, you can see the request header named 'Authorization' with the value of ${access_token} after prefix 'Bearer '.

Reply:
fixed.